maven_iuvs/echelle.py

```diff
@@ -2261,15 +2264,14 @@ def loglikelihood(params, wavelength_data, binedges, CLSF, data, uncertainty, s,
         A single value which represents either the log-likelihood (if negative)
         or the fit "badness" if positive.
     """
-    s = s / abs(s)  # Enforce s to be -1 or 1 since python doesn't have sign.
 
     # Now do the model 
     DN_fit, *_ = lineshape_model(params, wavelength_data, binedges, CLSF, BU_bg) 
 
     # Fit the model to the existing data assuming Gaussian distributed photo events 
     L = np.sum((data - DN_fit)**2 / (2*(uncertainty**2) ) )
```

Comment:
I hate to prolong this discussion, but I think loglikelihood needs to have a minus sign in it here, which gets cancelled out by negloglikelihood. (Will also need to update which function is called by scipy minimize vs dynesty)

Reply:
You forced me to review my obs and stats notes but you're right, the LL for a Gaussian has a minus sign. This makes it frustratingly unintuitive but we are stuck with it.

Reply:
The way it makes intuitive sense to me is that log likelihood represents the log of a probability, and probabilities are always smaller than one, so the log is negative.

Approving!

Reply:
hadn't yet pushed updates when you wrote most recent comments. All are now fixed in 3bad7013aae1dda5b1104b64b052c651dbb69643. Please review and check that it's all sorted!